deploy/aws/tidb-cluster/values/default.yaml

```diff
@@ -2,19 +2,39 @@
 timezone: UTC
 
 pd:
-  logLevel: info
+  config: |
+    [log]
+    level = "info"
+    [replication]
+    location-labels = ["region", "zone", "rack", "host"]
+
   storageClassName: ebs-gp2
 tikv:
-  logLevel: info
+  config: |
+    log-level = "info"
+
   stroageClassName: local-storage
   syncLog: true
 tidb:
-  logLevel: info
+  config: |
+    [log]
+    level = "info"
+
   service:
     type: LoadBalancer
     annotations:
       service.beta.kubernetes.io/aws-load-balancer-internal: '0.0.0.0/0'
       service.beta.kubernetes.io/aws-load-balancer-type: nlb
+  separateSlowLog: true
+  slowLogTailer:
+    image: busybox:1.26.2
+    resources:
+      limits:
+        cpu: 100m
+        memory: 50Mi
+      requests:
+        cpu: 20m
+        memory: 5Mi
```

Comment:
The modifications of this file are same as the default one in the chart's values.yaml, so it is unnecessary

Reply:
For the charts released till now, the configs for tidb/tikv/pd are different and separateSlowLog is commented out.

Reply:
Yes, the old version `v1.0.0-beta.3` don't need this because the chart has a proper default, and if we release your change, the new version neither need this because the new version will have a proper default too.

Moreover, keep these configurations here will increase our maintenance burden.

Reply:
OK, just keep the "separateSlowLog: true" because it's false in old charts and it's better to set it to true for the slow log.